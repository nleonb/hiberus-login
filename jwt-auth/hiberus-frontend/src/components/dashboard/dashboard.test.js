import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from "react-router-dom";

import Dashboard from './dashboard';

describe('Test Dashboard component ', () => {
    it('renders correctly', () => {
        const component = renderer.create(
            <MemoryRouter initialentries="{['/']}">
                <Dashboard />
            </MemoryRouter>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});