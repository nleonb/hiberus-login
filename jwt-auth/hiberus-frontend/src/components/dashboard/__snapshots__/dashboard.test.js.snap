// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Dashboard component  renders correctly 1`] = `
<div
  className="container mt-5"
>
  <h1>
    Welcome Back: 
    
  </h1>
  <table
    className="table is-striped is-fullwidth"
  >
    <thead>
      <tr>
        <th>
          No
        </th>
        <th>
          Name
        </th>
        <th>
          Email
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
